src/System.Data.Common/src/System/Data/DataTableExtensions.cs

```diff
@@ -12,7 +12,7 @@ namespace System.Data
     /// <summary>
     /// This static class defines the DataTable extension methods.
     /// </summary>
-    public static class DataTableExtensions
+    public static partial class DataTableExtensions
```

Comment:
Nit: do we still need to make types partial?

Reply:
No, I missed removing when I merged the source back together.  I’ll delete.